client/components/account-status/account-tools/styles.scss

```diff
@@ -0,0 +1,11 @@
+.account-tools {
+	padding-top: 16px;
```

Comment:
maybe we can also use `$gap` for this which is exactly 16px

Reply:
Thanks, I didn't know that! Updated in 681adad07.